### PR TITLE
Telink b95 new old patches

### PR DESCRIPTION
**Pull request by haiwentelink**
update the ske part , and update the flash.c and flash.h for flash_read_mid interface.

### CODE REVIEW

File: tlsr9/drivers/B95/usbhw.h
Comment (pulin1103) [inline tlsr9/drivers/B95/usbhw.h:1]: It will disable the enabled endpoints.

File: tlsr9/drivers/B95/usbhw.h
Comment (pulin1103) [inline tlsr9/drivers/B95/usbhw.h]: This will disable the enabled endpoints.

File: tlsr9/drivers/B95/usbhw.c
Comment (pulin1103) [inline tlsr9/drivers/B95/usbhw.c:34]: Why is this function commented?

File: tlsr9/drivers/B95/usbhw.h
Comment (haiwentelink) [reply tlsr9/drivers/B95/usbhw.h]: follow the driver in new ble sdk

File: tlsr9/drivers/B95/usbhw.c
Comment (haiwentelink) [reply tlsr9/drivers/B95/usbhw.c:34]: in the new ble sdk , the func  pm_wait_dig_module_power_stable is disapperared .

File: zephyr/module.yml
Comment (s07641069) [inline zephyr/module.yml]: change hash after merge of https://github.com/telink-semi/zephyr_hal_telink_b91_ble_lib/pull/15

File: tlsr9/ble/vendor/controller/b9x_bt.c
Comment (s07641069) [inline tlsr9/ble/vendor/controller/b9x_bt.c]: better to add "lib/include/" into CMake include list and revert this change

File: tlsr9/ble/common/utility.h
Comment (s07641069) [inline tlsr9/ble/common/utility.h]: don't leave commented code.
remove if not needed.

File: tlsr9/common/b9x_rf_power.c
Comment (s07641069) [inline tlsr9/common/b9x_rf_power.c]: after adding "lib/include/" into CMake include list use "rf/rf_common.h"
as well to avoid lots of #if - you can put "rf/rf_common.h" into "rf.h" or something like this

File: tlsr9/crypto/mbedtls/internal/compatibility/aes_alt.c
Comment (s07641069) [inline tlsr9/crypto/mbedtls/internal/compatibility/aes_alt.c]: As well "lib/include" can be removed from path

File: tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c
Comment (s07641069) [inline tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c]: same  "lib/include" can be removed from path and after #if can be reverted (simplified)

File: tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c
Comment (s07641069) [inline tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c]: why we can't use same path for all SoCs?

File: tlsr9/crypto/mbedtls/internal/ecp_alt_b9x_backend.c
Comment (s07641069) [inline tlsr9/crypto/mbedtls/internal/ecp_alt_b9x_backend.c]: same here
clear for all changes include paths to use same includes and avoid lots of #if

**Review by s07641069 — CHANGES_REQUESTED** [review]

File: tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c
Comment (haiwentelink) [reply tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c]: for the driver file changes , and we don't want to to adjust the drive file .

File: tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c
Comment (haiwentelink) [reply tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c]: will add the lib/include /pke into cmake list .

File: tlsr9/crypto/mbedtls/internal/compatibility/aes_alt.c
Comment (haiwentelink) [reply tlsr9/crypto/mbedtls/internal/compatibility/aes_alt.c]: will add the lib/include /ske into cmake list .

File: tlsr9/ble/common/utility.h
Comment (haiwentelink) [reply tlsr9/ble/common/utility.h]: will delete

File: tlsr9/common/b9x_rf_power.c
Comment (haiwentelink) [reply tlsr9/common/b9x_rf_power.c]: will add the lib/include /rf into cmake list .

File: tlsr9/ble/vendor/controller/b9x_bt.c
Comment (haiwentelink) [reply tlsr9/ble/vendor/controller/b9x_bt.c]: will add the lib/include into cmake list .

File: tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c
Comment (s07641069) [reply tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c]: ok

**Comment by andriy-bilynskyy** [comment]
PR seems fine. But plese squash your commits. There is no any reasons to keep middle sources version where builds are not tested.

**Comment by haiwentelink** [comment]
Hi andriy , for the middle changes , we tested every version , i don't understand , so can you point out which commit need to squash , from xxx to xxx ? I guess from b61e2250d056abba92d0a7dc8fd78d79264de2fd to 1568eebcf2fbe382f70f5b683afd33ac94cbd216 ? For some commit should not be squash which we need to do patch for next version .

**Comment by s07641069** [comment]
@haiwentelink 
merge https://github.com/telink-semi/zephyr_hal_telink_b91_ble_lib/pull/15 (approved) and after update in this PR hash of lib (do again force push to have 1 commit)

**Review by s07641069 — APPROVED** [review]

**Comment by haiwentelink** [comment]
already update the hash of the of libs .

**Pull request merged**
